scvi/train/_trainer.py

```diff
@@ -156,4 +156,9 @@ def fit(self, *args, **kwargs):
                 category=UserWarning,
                 message="One of given dataloaders is None and it will be skipped",
             )
+            warnings.filterwarnings(
```

Comment:
let's only filter it if the model passed to fit is an instance of:

https://github.com/YosefLab/scvi-tools/blob/a9da914764d311ff18b4b557b78d9550a3b940af/scvi/train/_trainingplans.py#L581

See the pylit API here, model should be the first arg.

https://pytorch-lightning.readthedocs.io/en/latest/common/trainer.html#fit